# a Texture1D's initial data is uploaded with the pitches given, as it was before
diff --git a/src/d3d11/d3d11_texture_device.cpp b/src/d3d11/d3d11_texture_device.cpp
index ac9dacf..596d375 100644
--- a/src/d3d11/d3d11_texture_device.cpp
+++ b/src/d3d11/d3d11_texture_device.cpp
@@ -463,7 +463,6 @@ HRESULT CreateDeviceTextureInternal(MTLD3D11Device *pDevice,
           uint32_t row, image, length;
           if (FAILED(GetLinearTextureLayout(pDevice, finalDesc, sub.MipLevel, row, image, length, false)))
             return E_FAIL;
-          data.SysMemPitch = data.SysMemSlicePitch = row;
         }
         initializer.initWithData(
             texture.ptr(), texture->current(), sub.ArraySlice, sub.MipLevel, data.pSysMem, data.SysMemPitch,
